src/DeploymentEnvironments.ts

```diff
@@ -156,6 +156,13 @@ export const deploymentEnvironments: { [key: string]: Configuration } = {
         description: 'EP role used!',
         priority: 'critical',
       },
+      {
+        roleMonitoring: {
+          roleName: 'landingzone-break-glass',
+        },
+        description: 'Break the glass role used!',
+        priority: 'critical',
+      },
```

Comment:
We krijgen al meldingen van het gebruik, hoe is deze anders?

Reply:
Deze geeft accountnaam en principal mee in slack. Het alarm dat nu af gaat is ook niet terug te vinden in de LZ repo. Daarom wil ik het hier oplossen zodat het CloudWatch alarm van deze en de normale EP rol verwijderd kunnen worden.